libakumuli/storage_engine/blockstore.cpp

```diff
@@ -377,10 +324,168 @@ static u32 crc32c(const u8* data, size_t size) {
     return impl(0, data, size);
 }
 
-u32 FixedSizeFileStorage::checksum(u8 const* data, size_t size) const {
+u32 FileStorage::checksum(u8 const* data, size_t size) const {
     return crc32c(data, size);
 }
 
+// FixedSizeFileStorage
+
+FixedSizeFileStorage::FixedSizeFileStorage(std::string metapath, std::vector<std::string> volpaths)
+  : FileStorage::FileStorage(metapath, volpaths)
+{
+  // nothing specific needed except calling the parent constructor
+}
+
+std::shared_ptr<FixedSizeFileStorage> FixedSizeFileStorage::open(std::string metapath, std::vector<std::string> volpaths) {
+  if (volpaths.empty() || metapath.empty()) {
```

Comment:
yes, I missed that one. My emacs setup didn't play nicely so manually fixed it up. I'll go over it and find any 2 spaces cases.